CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
   Workload Plane/Control Plane CIDRs.
   (PR[#4217](https://github.com/scality/metalk8s/pull/4217))
 
+- Add a new salt module `metalk8s_kubernetes_cronjob`
+  to manage Kubernetes Cronjobs objects.
+  (PR[#4240](https://github.com/scality/metalk8s/pull/4240))
+
 ### Bug fixes
 
 - Salt module metalk8s_monitoring fixed and improved
```

Comment:
Nit: Just to be consistent even if it's not in this PR can we update this one as well

```suggestion
- Salt module `metalk8s_monitoring` fixed and improved
```

Reply:
Sure, I wasn't cautious